plugins/woocommerce-admin/client/customize-store/transitional/index.tsx

```diff
@@ -127,115 +135,145 @@ export const Transitional = ( {
 					>
 						{ __( 'View store', 'woocommerce' ) }
 					</Button>
+
+					<Button
+						variant="primary"
+						onClick={ () => {
+							trackEvent(
+								'customize_your_store_transitional_home_click'
+							);
+							sendEvent( {
+								type: 'GO_BACK_TO_HOME',
+							} );
+						} }
+					>
+						{ __( 'Back to Home', 'woocommerce' ) }
+					</Button>
```

Comment:
I believe you wanted to keep this button behind the `isEntrepreneurFlow()` check, right? Because otherwise it loads on the normal flow as well.

Reply:
Great catch. Thank you!